keps/sig-auth/4412-projected-service-account-tokens-for-kubelet-image-credential-providers/README.md

```diff
@@ -1048,6 +1076,11 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+On failure to fetch credentials from an exec plugin, the kubelet will retry after some period and invoke the plugin again.
```

Comment:
Kubernetes has its own SLO.

https://github.com/kubernetes/community/tree/master/sig-scalability/slos

I think you should mention which one of these would be impacted if this feature misbehaves.

Reply:
From that doc

> We are explicitly excluding image pulling from time the SLI. This is because it highly depends on locality of the image, image registry performance characteristic (e.g. throughput), image size itself, etc. Since we have no control over any of those (and all of those would significantly affect SLI) we decided to simply exclude it.

Given that image pulling is excluded, do we need to make any other updates here?

Reply:
I don't think any of those SLOs apply, though pod startup might need a closer reading.

It makes sense that the actual image pull time should be excluded. I think the relevant case here would be outright failures?

Reply:
right, pod would fail to startup.

Reply:
In this case, I think we can explain why those SLOs don't apply but maybe mention the kind of issues you'd see here?

Reply:
Ty for adding that!